deployment/blobStorage.bicep

```diff
@@ -0,0 +1,66 @@
+
+param name string
+param location string = resourceGroup().location
+param sku string = 'Standard_RAGRS'
+param managedIdName string
+param otherIds array = []
+
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
+  name: managedIdName
+}
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-04-01' = {
+  name: '${name}storage'
+  location: location
+  kind: 'StorageV2'
+  sku: {
+    name: sku
+  }
+  properties: {
+    accessTier: 'Hot'
+    allowBlobPublicAccess: true
```

Comment:
I believe we do since the github action runners and the ACI's need access, access is still role based. Having said that, I see the default value for this is true so this line isn't actually necessary

Reply:
Removed in 1c39ab12f3d83c7664549684dafc55a31692af75